panel/models/fullcalendar.js

```diff
@@ -0,0 +1,104 @@
+import {Calendar} from "@fullcalendar/core"
```

Comment:
To be honest I don't believe this component is "core" Panel. Optimally this would be in a separate package like `panel-mermaid`.

It increases the complexity of the Panel project. And the download size which is important in for example Pyodide.

Reply:
> If loading plugins dynamically means loading modules dynamically then

I meant part of the compilation.

Is 136 KBs significant?
https://bundlephobia.com/package/@fullcalendar/core@6.1.15

Reply:
Its not the js I'm "worried" about. Its the panel python package.

Reply:
I'm also leaning towards splitting this out. That said:

>And the download size which is important in for example Pyodide

Pyodide bundles do not and will not ship with the JS bundles.

Reply:
I guess many people will get Panel directly from PyPI. We probably should finally figure out how to split that package, so `panel/dist` is shipped separately. For backward compatibility we'd probably have to create a new package called `panel-core` and then make `panel` a metapackage that depends on `panel-core` and additionally ships `panel/dist`.